### DISCUSSION TITLE
Is it possible to pass the language as a parameter for an URL?

### DISCUSSION THREAD

**Discussion by mapreri** [globaleaks/globaleaks-whistleblowing-software#4354] [Q&A] [answered]
### What version of GlobaLeaks are you using?

5.0.34

### What operating system(s) are you seeing the problem on?

N/A

### Describe the issue

There was this feature which let people reach a localized version with a defined URL, i.e.

http://demo.globaleaks.org/#/it
http://demo.globaleaks.org/#/en

this was done as part of https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/778

Apparently that is not working with current versions of globaleaks.  Was that feature retracted?  Any alternative?

**Comment by evilaliv3** [comment, marked as answer]
Hello @mapreri 

You could use ?lang=$lang for example ?lang=en for english and ?lang=it for italian

**Comment by mapreri** [comment]
Thank you for your answer.  I couldn't find this information anywhere, so it might be a good idea to document this better.

(also sorry for opening this as a bug... tbh this would be the first time for me to even use the "discussion" feature of github!)
